src/Compilers/CSharp/Portable/Symbols/Source/ParameterHelpers.cs

```diff
@@ -337,11 +337,11 @@ internal static MethodSymbol FindContainingGenericMethod(Symbol symbol)
             return null;
         }
 
-        private static RefKind GetModifiers(SyntaxTokenList modifiers, out SyntaxToken argPassingKeyword, out SyntaxToken paramsKeyword, out SyntaxToken thisKeyword)
+        private static RefKind GetModifiers(SyntaxTokenList modifiers, out SyntaxToken refOrOutKeyword, out SyntaxToken paramsKeyword, out SyntaxToken thisKeyword)
```

Comment:
>GetModifiers [](start = 31, length = 12)

Since now the ref kind is not expressed in a single token (might be two), I suggest changing the signature of this function to be:
        private static RefKind GetModifiers(SyntaxTokenList modifiers, out SyntaxToken keyword)
And use the returned RefKind to figure out the type and do error reporting on the returned token.
We can return the ReadOnly or the In keyword in case of ref readonly modifiers, because this is where we're going to report errors on.